examples/2d/contributors.rs

```diff
@@ -207,7 +203,7 @@ fn select(
     contributor: &Contributor,
     transform: &mut Transform,
     text: &mut Text,
-    name: &str,
+    #[allow(clippy::ptr_arg)] name: &String,
```

Comment:
Following the same logic as the `for_each` discussion above, I'm not sure if we should do this in example code.

Reply:
This is really just a clippy bug, so I don't think the code is any more difficult to understand. On that note, I filed https://github.com/rust-lang/rust-clippy/issues/8361